features/alphabetSearch/dataTables.alphabetSearch.js

```diff
@@ -49,36 +49,44 @@ $.fn.dataTable.ext.search.push( function ( context, searchData ) {
 		return true;
 	}
 
-    if ($.fn.dataTable.AlphabetSearch.caseSensitive) {
-        if ( searchData[$.fn.dataTable.AlphabetSearch.column].charAt(0) === context.alphabetSearch ) {
-            return true;
-        }
-    } else {
-        if ( searchData[$.fn.dataTable.AlphabetSearch.column].charAt(0).toUpperCase() === context.alphabetSearch ) {
-            return true;
-        }
-    }
+	if (context.oInit.alphabet !== undefined) {
```

Comment:
I wasn't sure how to get variables set in alphabet visible and this seemed like the only way to do it. If there's better way please let me know how.